README.md

```diff
@@ -1,92 +1,198 @@
+3. [Deploy](#3-deploy)
+4. [Bonus: Dev environments](#bonus-dev-environments)
+
+#### 1. Create your first datacenter
+
+DevOps, platform, and infrastructure teams can have all sorts of requirements for how applications need to behave inside their environments. This is why we created the [datacenters framework](./src/datacenters). This is mostly noise for developers who just want to focus on their app though, so we've create a set of example datacenters you can use to get started.
+
+The datacenter schema used below doesn't use a cloud provider at all. Instead, it runs your applications using docker for deployments and databases, traefik for your API gateway and service mesh, and the local filesystem to store secrets. That means it won't cost you a dime to run the applications and the data never leaves your device:
+
+```sh
+$ arcctl apply datacenter local https://raw.githubusercontent.com/architect-team/arcctl/main/examples/datacenters/local.yml
 ```
 
-## Cloud Providers
+Datacenters are designed to be a home for many environments, now and in the future. You can leave your datacenter running indefinitely to make it easier to create new test environments.
+
+#### 2. Create an environment
 
-Before you'll be able to interact with any cloud resources, you'll need to register your cloud credentials with arcctl:
+Once you've created a datacenter, you'll need to create an [environment](./src/environments) that you can deploy into. Environments are basically namespaces that can allow datacenters to power more than one environment (populate for non-production use cases). Go ahead and create one on the datacenter you just created:
 
 ```sh
-# The CLI will prompt you for available provider types and required credentials
-$ arcctl add account
+$ arcctl create environment my-env --datacenter local
 ```
 
-Scroll down to see more information about how to register each provider. **We highly recommend taking the time to read through our Readme for your desired cloud provider.** While arcctl can help to simplify the managment of resources, each cloud provider has it's own quirks that can still add some complexity to getting started. Our step by step guide can help make sure everything wroks smoothly the first time.
+#### 3. Deploy
 
-### Supported providers
-- [x] [digitalocean](./src/%40providers/digitalocean/)
-- [x] [aws](./src/%40providers/aws/)
-- [ ] gcp
-- [ ] azure
+Finally we can test out deploying a component! We've curated a sample application designed to show off the power of dependencies, [architect-team/auth-example](https://github.com:architect-team/auth-example). This component is a full-stack web application that connects our [Ory Kratos component](https://github.com/architect-team/kratos-selfservice-ui-node) to handle user registration and login, which in turn connects to our [Mailslurper component](https://github.com/architect-team/mailslurper) to handle SMTP for email verification. The component architecture looks a bit like this:
```

Comment:
A plantuml diagram may be huge here to help showcase what is where and why. Especially if the developer has never heard of these before.  You kind of have it below but putting inline descriptions into the graph could be nice. Would also break up the wall of text.

Can help make it if you need.